docusaurus.config.js

```diff
@@ -11,6 +11,9 @@ const docusaurusConfig = {
   organizationName: 'saucelabs',
   projectName: 'sauce-docs',
   scripts: [
+    {
+      src: 'src="https://gist.github.com/discombobulateme/7dbf32500790c00ce6a398a6ff06cf9f.js'
+    },
```

Comment:
@discombobulateme , is the 'src=' required for this particular script?

Reply:
🤦🏻‍♀️ hahah no, love code reviews...correcting it!